ui/selectors/selectors.test.js

```diff
@@ -738,4 +739,20 @@ describe('Selectors', () => {
     mockState.metamask.snapsInstallPrivacyWarningShown = null;
     expect(selectors.getSnapsInstallPrivacyWarningShown(mockState)).toBe(false);
   });
+
+  describe('#getAllowedAnnouncementIds', () => {
+    it('includes properties for all notifications exported from shared/nofitications/index.js', () => {
+      const allowedAnnouncements =
+        selectors.getAllowedAnnouncementIds(mockState);
+
+      const uiNotificationIds = Object.keys(UI_NOTIFICATIONS);
+
+      const uiNotificationIdsNotInAllowedAnnouncements =
+        uiNotificationIds.filter(
+          (id) => allowedAnnouncements[id] === undefined,
+        );
+
+      expect(uiNotificationIdsNotInAllowedAnnouncements).toStrictEqual([]);
```

Comment:
what if we used [`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)?:

```
const hasMissingNotificationId = uiNotificationIds.filter(
          (id) => allowedAnnouncements[id] === undefined,
);

expect(hasMissingNotificationId).toBe(true)
```